mk-model.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
The model file will actually be created automatically for you if the containers are shut down in the correct way, and the correct order, however I don't think the stop script currently does this correctly so let's leave this script here for now but will remove in near future.

Reply:
Sounds good.